tests/UnitTests/ValidatableExtensions/ValidatableExtensions.StringPropertiesTests.cs

```diff
@@ -522,4 +522,112 @@ public void ThrowIfPropertyNotStartsWith_WhenPropertyStartsWith_ShouldNotThrow()
         // Assert
         action.Should().NotThrow();
     }
+
+    [TestMethod]
+    public void ThrowIfPropertyContains_WhenPropertyContains_ShouldThrow()
+    {
+        // Arrange
+        var person = new { Name = "John" };
+
+        // Act
+        Action action = () => person.Throw().IfContains(p => p.Name, "oh");
+
+        // Assert
+        action.Should().ThrowExactly<ArgumentException>()
+            .WithMessage($"String should not contain 'oh' (Comparison Type: Ordinal). (Parameter '{nameof(person)}: p => p.Name')");
+    }
+
+    [TestMethod]
+    public void ThrowIfPropertyContains_WhenPropertyNotContains_ShouldNotThrow()
+    {
+        // Arrange
+        var person = new { Name = "John" };
+
+        // Act
+        Action action = () => person.Throw().IfContains(p => p.Name, "Oh");
+
+        // Assert
+        action.Should().NotThrow();
+    }
+
+    [TestMethod]
+    public void ThrowIfPropertyNotContains_WhenPropertyNotContains_ShouldThrow()
+    {
+        // Arrange
+        var person = new { Name = "John" };
+
+        // Act
+        Action action = () => person.Throw().IfNotContains(p => p.Name, "Oh");
+
+        // Assert
+        action.Should().ThrowExactly<ArgumentException>()
+            .WithMessage($"String should contain 'Oh' (Comparison Type: Ordinal). (Parameter '{nameof(person)}: p => p.Name')");
+    }
+
+    [TestMethod]
+    public void ThrowIfPropertyNotContains_WhenPropertyContains_ShouldNotThrow()
+    {
+        // Arrange
+        var person = new { Name = "John" };
+
+        // Act
+        Action action = () => person.Throw().IfNotContains(p => p.Name, "oh");
+
+        // Assert
+        action.Should().NotThrow();
+    }
+
+    [TestMethod]
+    public void ThrowIfPropertyContains_UsingOrdinalIgnoreCase_WhenPropertyContains_ShouldThrow()
```

Comment:
Small change following MSDN's [best practices](https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#naming-your-tests) (here and below)
```suggestion
    public void ThrowIfPropertyContains_WhenPropertyContainsUsingOrdinalIgnoreCaseComparison_ShouldThrow()
```